code/Collector/DatabaseCollector.php

```diff
@@ -173,8 +173,8 @@ public function getWidgets()
     public function getAssets()
     {
         return array(
-            'base_path' => '/' . DEBUGBAR_DIR . '/javascript',
-            'base_url' => DEBUGBAR_DIR.'/javascript',
+            'base_path' => '/' . Director::makeRelative(ModuleLoader::getModule('lekoala/silverstripe-debugbar')->getResource('javascript')->getURL()),
```

Comment:
Do you think you could consolidate this repeated config into a DebugBarConfig or something?